frontend/src/components/Terminal.tsx

```diff
@@ -21,6 +22,12 @@ const Terminal = () => {
     const [driver, setDriver] = useState(new CQLDriver());
     const classes = useStyles();
 
+    const [adress, setAddress] = useState("");
+    const [port, setPort] = useState<string>("");
```

Comment:
Może szczegół
Ale czemu ten jeden ma podany explicite typ?

Reply:
Bo był dodany w innym momencie niz inne, to bez roznicy jest, mozna zmienic jak cchesz zwiekszyc consistency.